changelogs/unreleased/6501-fix-error-message-wrong-type.yml

```diff
@@ -0,0 +1,4 @@
+description: Fix a wrong error message when using a wrong type in the inmanta language.
```

Comment:
I don't get this description. As far as I can see this PR only makes sure that the types are represented using a small leading character instead of a capital letter (Bool becomes bool for example). Am I missing something?

Reply:
indeed I created this description before the fix was already done. I'll change it;